src/packer.pkr.hcl

```diff
@@ -1,3 +1,10 @@
+variable "admin_pass" {
```

Comment:
Please update the README to include this variable's information. I can walk you through how I handled doing it when we converted to an HCL2 template if you'd like.

I would also like to propose we rename this variable. This is specifically being used in this configuration for WinRM connectivity (so maybe `winrm_password`?) and I think it should be named as such. I would also like the `winrm_username` value turned into a variable with a default of `Administrator` for completeness.

Reply:
That would be great. Let me know when you're available to walk me through it. 

Well this password has two purposes in this case, to be set by WinRM as the Windows Administrator password (refer to the user data file) and when its been set, packer well then connect to WinRM with it. But i see where you're coming from. I just figured it'd be clearer if its known that this is also the password you'd need to use to login to windows with.

Reply:
resolved with [bbc49e9](https://github.com/cisagov/windows-server-packer/pull/10/commits/bbc49e9151dd84e85cb328398c205adc953440ce)

Reply:
@mcdonnnj resolved with [5359bf1](https://github.com/cisagov/windows-server-packer/pull/10/commits/5359bf1eefb309497367e3510b420ecf2fecb824)

Reply:
@mcdonnnj resolved with [4e505f6](https://github.com/cisagov/windows-server-packer/pull/10/commits/4e505f61b1e97d54725d8ffe0d86b453dfa88246)